### DISCUSSION TITLE
Unable to use nausea

### DISCUSSION THREAD

**Discussion by A3148475824** [Tampermonkey/tampermonkey#1619] [General]
![Screenshot_2022-10-14-15-00-58-0941587243](https://user-images.githubusercontent.com/70214970/195783795-3cdfde2c-fd20-4695-9300-78f9ff639e93.png)
Clicking on the extension does nothing!  What about extensions?  What about the switch?  Click to close the script and it all disappears, how can it be so bad!

**Comment by derjanb** [comment]
Duplicate of #1607
